src/current/v23.1/log-sql-statistics-to-datadog.md

```diff
@@ -0,0 +1,90 @@
+   1. To connect to Datadog, replace `{DATADOG API KEY}` with the value you copied in Step 1.
+   1. To control the ingestion and potential drop rate for sampled query events, configure the following `buffering` values depending on your workload:
+
+   - `max-staleness`: The maximum time a log message will wait in the buffer before a flush is triggered. Set to `0` to disable flushing based on elapsed time. Default: `5s`
+   - `flush-trigger-size`: The number of bytes that will trigger the buffer to flush. Set to `0` to disable flushing based on accumulated size. Default: `1MiB`. In this example, override to `2.5MiB`.
+   - `max-buffer-size`: The maximum size of the buffer: new log messages received when the buffer is full cause older messages to be dropped. Default: `50MiB`
+
+{% include_cached copy-clipboard.html %}
+~~~ yaml
+sinks:
+  http-servers:
+    datadog:
+      channels: [TELEMETRY]
+      address: https://http-intake.logs.datadoghq.com/api/v2/logs
+      format: json
+      method: POST
+      compression: gzip
+      headers: {DD-API-KEY: "{DATADOG API KEY}"} # replace with actual API key
+      buffering:
+        format: json-array
+        max-staleness: 5s
+        flush-trigger-size: 2.5MiB # override default value
+        max-buffer-size: 50MiB
+  file-groups: # override default configuration
+    telemetry:  # do not write telemetry events to disk
+      channels: [] # set to empty square brackets
+~~~
+
+Pass the [`logs.yaml` file]({% link {{ page.version.version }}/configure-logs.md %}#yaml-payload) to the  `cockroach` process with either `--log-config-file` or ` --log` flag.
+
+## Step 3. Configure CockroachDB to emit query events
+
+Enable the [`sql.telemetry.query_sampling.enabled` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-telemetry-query-sampling-enabled) so that executed queries will emit an event on the telemetry [logging channel]({% link {{ page.version.version }}/logging-overview.md %}#logging-channels):
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING sql.telemetry.query_sampling.enabled = true;
+~~~
+
+Set the `sql.telemetry.query_sampling.max_event_frequency` cluster setting to `100000` to emit query events at a higher rate per second than the default value of `8`, which is extremely conservative for the Datadog HTTP API. This cluster setting controls the max event frequency at which CockroachDB samples queries for telemetry.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING sql.telemetry.query_sampling.max_event_frequency = 100000;
+~~~
+
+## Step 4. Monitor TELEMETRY logs in Datadog
+
+1. Navigate to [**Datadog > Logs**](https://app.datadoghq.com/logs).
+1. Filter by **OTHERS > channel: TELEMETRY** to see the logs for the query events that are emitted. For example:
+
+<img src="{{ 'images/v23.1/datadog-telemetry-logs.png' | relative_url }}" alt="Datadog Telemetry Logs" style="border:1px solid #eee;max-width:100%" />
```

Comment:
Just a suggestion, but "See also" section might be good to end the page

Reply:
Added "See also" section.